rocketpy/tools.py

```diff
@@ -1231,6 +1231,42 @@ def find_closest(ordered_sequence, value):
     return pivot_index - 1 if value - smaller <= greater - value else pivot_index
 
 
+def import_optional_dependency(
+    name,
+):
```

Comment:
For some reason, `black` does not flag this when linting. I don't think there is any need to break the line here.

```suggestion
def import_optional_dependency(name):
```

Reply:
Solved: [7cf2435](https://github.com/RocketPy-Team/RocketPy/pull/403/commits/7cf2435ed069f331608bafd471081888d3e40ad4)